yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-index-usage.psql

```diff
@@ -0,0 +1,55 @@
+CREATE TEMP TABLE temp_table AS
```

Comment:
Maybe we can add a general comment explaining the various things we are doing in this query? Might make it easier to understand for future references?